src/controllers/connectionManager.ts

```diff
@@ -597,6 +597,22 @@ export default class ConnectionManager {
         }
     }
 
+    public onDidRenameTextDocument(oldFileUri: string, newFileUri: string): void {
```

Comment:
>public onDidRenameTextDocument(oldFileUri: string, newFileUri: string): void [](start = 4, length = 76)

this looks basically the same as onUntitledFileSaved.  Can we add a new method and make both the event handlers call a single implementation of this logic?